bellboy/actors/generic.py

```diff
@@ -15,9 +15,26 @@ def __init__(self, *args, **kwargs):
         # private attributes
         self.parent = None
         self.log = None
+        self._address_book = {}
         self.status = Response.NOT_READY
         self.TEST_MODE = False
 
+    def nameOf(self, address: ActorAddress):
+        """
+        Returns name of actor if it's entry exists in address book.
+        Else returns the toString of the actorAddress.
+        """
+
+        return self._address_book.get(str(address), str(address))
+
+    def _nameAddress(self, address: ActorAddress, name: str):
+        """
+        Adds entry to my address book.
+        """
+        if name is not None:
+            # using the str of the address as key bc the address itself is unhashable type
+            self._address_book[str(address)] = name
+
     def createActor(
```

Comment:
Is this overridden/called when an actor is created @yozohu ?

Reply:
Yeah, this createActor is a wrapper for thespians actor.createActor (not actorSystem.createActor tho).  
It creates the actor, and configures its logging to fit our system, and adds its global name to relevant addressbooks.

Reply:
i should add a comment in the code, mb